code/datums/outfits/outfit_admin.dm

```diff
@@ -539,63 +539,153 @@
 	if(istype(P))
 		P.attack_self(H) // activate them, display musical notes effect
 
+// Soviet Military
+
 /datum/outfit/admin/soviet
 	name = "Soviet Generic"
 	gloves = /obj/item/clothing/gloves/combat
 	uniform = /obj/item/clothing/under/soviet
 	back = /obj/item/storage/backpack/satchel
-	head = /obj/item/clothing/head/ushanka
+	head = /obj/item/clothing/head/sovietsidecap
 	id = /obj/item/card/id
+	shoes = /obj/item/clothing/shoes/combat
 
 /datum/outfit/admin/soviet/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	. = ..()
 	if(visualsOnly)
 		return
 
+	if(H.gender == MALE)
+		H.real_name = capitalize(pick(GLOB.first_names_russian_male)) + " " + capitalize(pick(GLOB.last_names_russian_male))
+	else
+		H.real_name = capitalize(pick(GLOB.first_names_russian_female)) + " " + capitalize(pick(GLOB.first_names_russian_female))
+	H.name = H.real_name
+
 	var/obj/item/card/id/I = H.wear_id
 	if(istype(I))
 		apply_to_card(I, H, list(ACCESS_MAINT_TUNNELS), name)
+	H.sec_hud_set_ID()
 
 /datum/outfit/admin/soviet/tourist
 	name = "Soviet Tourist"
 
-	gloves = /obj/item/clothing/gloves/color/black
-	shoes = /obj/item/clothing/shoes/black
+	gloves = null
 	l_ear = /obj/item/radio/headset
 	backpack_contents = list(
 		/obj/item/storage/box/survival = 1
 	)
 
+/datum/outfit/admin/soviet/conscript
+	name = "Soviet Conscript"
+
+	gloves = null
+	l_ear = /obj/item/radio/headset/syndicate
```

Comment:
```suggestion
	l_ear = /obj/item/radio/headset/syndicate
```
Move this to the base `/datum/outfit/admin/soviet` and you can delete it in all the other outfits